### DISCUSSION TITLE
New firebase

### DISCUSSION THREAD

**Discussion by DevVigilante** [Sketchware-Pro/Sketchware-Pro#671] [Q&A] [answered]
I want to use latest firebase library but I can't do that using Library manager please help me to change firebase version for my project

**Comment by j-px** [comment, marked as answer]
You can't currently, see #696 for details.

**Reply by j-px** [reply]
Update: You now can. Add specific Firebase libraries via Local library Manager, then exclude built-in ones of potentially older versions.
